boards/nrf52840dongle/include/periph_conf.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2019 Christian Amsüss <chrysn@fsfe.org>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_nrf52840dongle
+ * @{
+ *
+ * @file
+ * @brief       Peripheral configuration for the nRF52840-Dongle
+ *
+ * @author      Christian Amsüss <chrysn@fsfe.org>
+ *
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include "periph_cpu.h"
+#include "cfg_clock_32_1.h"
+#include "cfg_rtt_default.h"
+#include "cfg_timer_default.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    UART configuration
+ *
+ * This board does not have explicit UART pins, but until a USB TTY can
+ * serve as a serial console, not defining any UART makes examples that build a shell much harder.
+ *
```

Comment:
You can drop that comment now :wink:

Reply:
Good catch, fixed :-)

(The UART configuration is left in place, in alignment with doc.txt and general debug / fallback usage now advertised in that place).